standard/standard_8.php

```diff
@@ -1090,5 +1090,6 @@ function array_slice(array $array, int $offset, ?int $length, bool $preserve_key
 #[Pure]
 function array_merge(
     #[PhpStormStubsElementAvailable(from: '5.3', to: '7.3')] $array,
-    array ...$arrays
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $arrays,
+    #[PhpStormStubsElementAvailable(from: '7.1')] array ...$arrays
```

Comment:
1) There's weird result for 5.4-5.5: https://3v4l.org/Y0Ggj#v540:
Parameter name: string(4) "arr1"
Parameter name: string(4) "arr2"
Parameter name: string(3) "...". It is optional. 
We should probably add it?

Reply:
I don't think it makes sense as 5.4-5.5 are too old versions and complication of stubs for the sake of 2 almost not used versions doesn't worth it.